examples/anydata-type/anydata_type.bal

```diff
@@ -1,30 +1,22 @@
 import ballerina/io;
 
-// This function returns a value of the `anydata` type.
-function getValue() returns anydata {
-    string name = "cat";
-    return name;
-}
-
 public function main() {
-    // In this example, the variable named `a` of the `anydata` type holds an `int` value.
-    anydata a = 5;
-    io:println(a);
+    // In this example, the variable named `data1` of the `anydata` type holds an `int` value.
+    anydata data1 = 5;
+    io:println(data1);
 
-    // Before using the value of `a` in arithmetic operations, we need to
-    // ascertain that it is indeed an `int`. A type cast or
-    // a type guard can be used for this.
-    int intVal = <int>a;
+    // Before using the value of `data1` in arithmetic operations, we need to ascertain 
+    // that it is indeed an `int`. A type cast or a type guard can be used for this.
```

Comment:
```suggestion
    // that it is an actual `int`. A type cast or a type guard can be used for this.
```

Reply:
The meaning here should be more like "that is actually an `int`".

Reply:
ok +1